plugins/scaffolder/src/components/MultistepJsonForm/schema.ts

```diff
@@ -21,7 +21,26 @@ function isObject(value: unknown): value is JsonObject {
   return typeof value === 'object' && value !== null && !Array.isArray(value);
 }
 
+function extractTopLevelUiSchema(schema: JsonObject, uiSchema: JsonObject) {
```

Comment:
Would prefer to refractor the existing recursive function to just lift over the props at the top level instead of both top and next level. This kinda ends up computing for depth n and n + 1, then n +1 and n + 2, etc.